dev/benchmarks/platform_views_layout/test_driver/scroll_perf_test.dart

```diff
@@ -7,17 +7,17 @@ import 'package:test/test.dart' hide TypeMatcher, isInstanceOf;
 
 void main() {
   group('scrolling performance test', () {
-    FlutterDriver driver;
+    FlutterDriver? driver;
```

Comment:
nit: late

Reply:
The tear-down is doing a null-check before closing the driver. We'd have to remove that and assume that the driver is never null if we make it late here.

Reply:
`connect` can't return null with NNBD.

The only problem you could run into is connect throwing, in which case I don't think it even matters if we're in a bad state by tear down.